<?xml version="1.0"?>
<launch>


  <!-- start camera stream -->
  <include file="$(find video_stream_opencv)/launch/camera.launch">
    <arg name="video_stream_provider" value="0"/>
  </include> 

  <!-- pivot observation node start -->
  <node  pkg="koopman_pivot" type="pivot_observer.py" output="screen" name="pivot_observer"/>

  <!-- rviz -->
  <node  pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find koopman_pivot)/config/pivot.rviz"/>

</launch>
